exercises/pig-latin/.meta/src/reference/java/PigLatinTranslator.java

```diff
@@ -21,11 +21,17 @@ String translate(String sentence) {
                 .collect(Collectors.joining(" "));
     }
 
+    private static int yLocation;
+
     private String translateWord(String word) {
         if (wordStartsWithVowelLike(word)) {
             return word + AY;
         }
 
+        if(hasYAfterConsonantCluster(word)){
```

Comment:
There should be white space after `if` and before `{`, like the other `if` statements already in this file 🙂